tests/integration/detection/test_integration_detection.py

```diff
@@ -75,7 +75,6 @@ def test_03_notebook_run(detection_notebooks):
     nb_output = sb.read_notebook(OUTPUT_NOTEBOOK)
     training_losses = nb_output.scraps["training_losses"].data
     assert len(training_losses) == epochs
-    assert training_losses[-1] < 1.5
```

Comment:
I added this test today. I normally passes but because of statistical non-deterministic effects sometimes does not. Hence removing now again.